src/main/java/org/hmhb/county/CountyDao.java

```diff
@@ -0,0 +1,14 @@
+package org.hmhb.county;
+
+import java.util.List;
+
+import org.springframework.data.repository.CrudRepository;
+
+/**
+ * Database access layer for {@link County} objects.
+ */
+public interface CountyDao extends CrudRepository<County, Long> {
+
+    List<County> findByNameStartingWithIgnoreCase(String namePart);
```

Comment:
Using JPA magic from the name of this method.

Keywords: http://docs.spring.io/spring-data/jpa/docs/1.5.1.RELEASE/reference/html/jpa.repositories.html#jpa.query-methods.query-creation